website/static/js/pages/login-page.js

```diff
@@ -1,6 +1,25 @@
 /**
  * Login page
  */
-var LogInForm = require('js/signIn');
+var $ = require('jquery');
 
+var LogInForm = require('js/signIn');
 new LogInForm.SignIn('#logInForm');
+
+var SignUp = require('../signUp.js');
+new SignUp('#signUpScope', '/api/v1/register/', $('#campaign').val());
```

Comment:
In general we discourage hard-coding URLs inside JS files. It limits both reusability and maintainability of the code.